web/static/js/explore.js

```diff
@@ -252,6 +252,72 @@ $.fn.dataTable.Api.register( 'clearPipeline()', function () {
   } );
 } );
 
+
+/**
+ * Function fetches log data based on params specified by user and download data in specified fileType
+ * @param {*} api_endpoint : API endpoint URL
+ * @param {*} column_list : List of Columns for the selected event type
+ * @param {*} api_params  : GET parameters for the API call
+ * @param {*} fileType    : File Type ( Json, Excel, Csv)
+ */
+function downloadLogData(api_endpoint, column_list, api_params, fileType) {
+    $.ajax({
```

Comment:
Data is fetched directly from the api rather than the table.

Reply:
@Ali-Razmjoo. The buttons are added inside the table. But on choosing on one. Data is fetched from api based on user filter and then converted to appropriate format and downloaded.